indexer/services/comlink/src/controllers/api/v4/vault-controller.ts

```
@@ -461,7 +463,7 @@ function getPnlTicksWithCurrentTick(
     return [];
   }
   const currentTick: PnlTicksFromDatabase = {
-    ...pnlTicks[pnlTicks.length - 1],
+    ...(_.maxBy(pnlTicks, 'blockTime')!),
```

Biome

[error] Forbidden extra non-null assertion.

Safe fix: Remove extra non-null assertion.


(lint/suspicious/noExtraNonNullAssertion)